models/models.go

```diff
@@ -20,6 +20,12 @@ const (
 	StepRunStatusCodeSkippedWithRunIf = 4
 	// StepRunStatusCodePreparationFailed ...
 	StepRunStatusCodePreparationFailed = 5
+	// StepRunStatusAborted ...
+	StepRunStatusAborted = 6
```

Comment:
This seems unused, what does this mean?

Reply:
This is supposed be the case when the CLI is aborted from outside. It is not used, as we exit immediately in this case: https://github.com/bitrise-io/bitrise/blob/086ce3c6ec13b5b5a4bf6b63bda67bb583398136/tools/timeoutcmd/timeoutcmd.go#L127.

Reply:
This is supposed be the case when the CLI is aborted from outside. It is not used, as we exit immediately in this case: https://github.com/bitrise-io/bitrise/blob/086ce3c6ec13b5b5a4bf6b63bda67bb583398136/tools/timeoutcmd/timeoutcmd.go#L127. Removed.